exporters/variables-scss/src/content/generators/contentGenerator.ts

```diff
@@ -0,0 +1,57 @@
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
+import { exportConfiguration } from '../../index';
+import { generateCssFromTokens, generateCssObjectFromTokens } from './cssGenerator';
+import { formatCSS } from '../formatters/cssFormatter';
+
+// Add disclaimer to the top of the content
+const addDisclaimer = (content: string): string => {
+  if (exportConfiguration.generateDisclaimer) {
+    return `/* This file was generated by Supernova, don't change manually */\n${content}`;
+  }
+
+  return content;
+};
+
+export const generateFileContent = (
+  tokens: Token[],
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  tokenTypes: TokenType | TokenType[],
+  groupNames: string | string[],
+  withCssObject: boolean,
+  hasParentPrefix: boolean,
+) => {
+  let cssTokens = '';
+  let cssObject = '';
+
+  // If tokenTypes or groupNames are strings, convert them to arrays
+  const tokenTypesArray = Array.isArray(tokenTypes) ? tokenTypes : [tokenTypes];
```

Comment:
DRY alert: What about creating a nice helper function to do this? Like `transform2Array`.

Reply:
outdated. Deleted